src/components/radio/README.md

```diff
@@ -54,6 +54,7 @@ The `md-radio-group` component has no button initially selected.
 | `value` | `any` | The value of this radio button. |
 | `checked` | `boolean` | Whether the radio is checked. |
 | `disabled` | `boolean` | Whether the radio is disabled. |
+| `actionName` | `string` | Used to set the `aria-label` attribute of the underlying input element. |
```

Comment:
This should be `aria-label`. We just want to forward the attribute. 

- As same as in the slide-toggle